images/metalk8s-keepalived/keepalived.conf.j2

```diff
@@ -0,0 +1,28 @@
+global_defs {
+    enable_script_security
+}
+
+vrrp_script check_ingress {
+    script "/etc/keepalived/check-ingress.sh"
+    interval 5
+    weight 60
+}
+
+{%- for data in input %}
+
+{%- set is_master = data.get("node") == env["NODE_NAME"] %}
```

Comment:
So, IIUC, an entry without a `node` being defined will not have a single MASTER. What's the effect of this?

Reply:
Indeed, just mean that he will go to one keepalived, a bit random

But if your question is if this VIP will keep moving between keepalived, the answer is no the VIP will go on one keepalived and stay on this one until a MASTER for this IP arrive or the keepalived hosting the VIP is no longer available (or lower priority)